vercel.json

```diff
@@ -1,5 +1,8 @@
 {
   "cleanUrls": true,
   "trailingSlash": false,
-  "redirects": [{ "source": "/chat", "destination": "https://discord.gg/grF4GTXXYm" }]
+  "redirects": [
+    { "source": "/chat", "destination": "https://discord.gg/grF4GTXXYm" },
+    { "source": "/", "destination": "/getting-started" }
```

Comment:
Would it be possible to add a 404 redirect to `/404`?

Reply:
I'll look into, not sure if we can do that without hard-coding all the existing paths.

Reply:
Ok, I'll look too - I'd expect something like `{ "source": "*", "status": "404", "destination": "/404" }` to work 🤔

Reply:
Hmm it looks like this might be working out of the box, nvm. See https://docs.astro.build/this-page-does-not-exist/

Reply:
404 should just work according to https://vercel.com/guides/custom-404-page#static-site-generatator-(ssg)